x/oracle/types/genesis.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	fmt "fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
+)
+
+func NewGenesisState(port string, sequence uint64, oracle string) *GenesisState {
+	return &GenesisState{
+		PortId:   port,
+		Sequence: sequence,
+		Oracle:   oracle,
+	}
+}
+
+// DefaultGenesis returns the default oracle genesis state
+func DefaultGenesis() *GenesisState {
+	return NewGenesisState(PortID, 1, "")
+}
+
+// Validate performs basic genesis state validation returning an error upon any
+// failure.
+func (gs GenesisState) Validate() error {
+	if gs.Sequence == 0 {
```

Comment:
This `Sequence` value is the "last sequence value used" right? If that's the case, it seems like a value of `0` should be allowed.

Reply:
I actually don't think the sequence is needed in state anymore. Good catch.